src/DIRAC/Core/Utilities/ElasticSearchDB.py

```diff
@@ -265,9 +267,15 @@ def updateDoc(self, index: str, docID: str, body) -> dict:
         """
         sLog.debug(f"Updating document {docID} in index {index}")
         try:
-            return S_OK(self.client.update(index, docID, body))
+            self.client.update(index, docID, body)
+        except ConflictError:
+            # updates are rather "heavy" operations from ES point of view, needing seqNo to be updated.
+            # Not ideal, but we just wait and retry.
+            time.sleep(3)
+            self.client.update(index, docID, body, retry_on_conflict=3)
```

Comment:
Is this any better than using the `retry_on_conflict` option and letting the server take care of retrying?

Reply:
I tried to find somewhere the information on if there's or not a delay in retrying, which I did not find. So I added it artificially (ugly, I agree).

Reply:
Is there any reason to delay? Naively I would have thought you could retry immediately.

Reply:
IIRC, and if things have not changed in the meanwhile, ES (and OpenSearch too I guess) for each update deletes the old document and then indexes a new one. The `seqNo` that is mentioned in the callstack keeps track of the version of the document. So basically an update is potentially 3 times slower than adding a new document (I am of course simplifying...). 

If we wait we have more chance that the previous update finished. I am anyway not even convinced this is needed...